dpe/src/commands/derive_child.rs

```diff
@@ -15,36 +15,122 @@ pub struct DeriveChildCmd {
     handle: ContextHandle,
     data: [u8; DPE_PROFILE.get_hash_size()],
     flags: u32,
-    tcb_type: u32,
+    tci_type: u32,
     target_locality: u32,
 }
 
 impl DeriveChildCmd {
-    const _INTERNAL_INPUT_INFO: u32 = 1 << 31;
-    const _INTERNAL_INPUT_DICE: u32 = 1 << 30;
-    const _RETAIN_PARENT: u32 = 1 << 29;
-    const _TARGET_IS_DEFAULT: u32 = 1 << 28;
+    const INTERNAL_INPUT_INFO: u32 = 1 << 31;
+    const INTERNAL_INPUT_DICE: u32 = 1 << 30;
+    const RETAIN_PARENT: u32 = 1 << 29;
+    const MAKE_DEFAULT: u32 = 1 << 28;
+    const CHANGE_LOCALITY: u32 = 1 << 27;
 
-    const fn _is_internal_input_info(&self) -> bool {
-        self.flags & Self::_INTERNAL_INPUT_INFO != 0
+    const fn uses_internal_info_input(&self) -> bool {
+        self.flags & Self::INTERNAL_INPUT_INFO != 0
     }
 
-    const fn _is_internal_input_dice(&self) -> bool {
-        self.flags & Self::_INTERNAL_INPUT_DICE != 0
+    const fn uses_internal_dice_input(&self) -> bool {
+        self.flags & Self::INTERNAL_INPUT_DICE != 0
     }
 
-    const fn _is_retain_parent(&self) -> bool {
-        self.flags & Self::_RETAIN_PARENT != 0
+    const fn retains_parent(&self) -> bool {
+        self.flags & Self::RETAIN_PARENT != 0
     }
 
-    const fn _is_target_is_default(&self) -> bool {
-        self.flags & Self::_TARGET_IS_DEFAULT != 0
+    const fn makes_default(&self) -> bool {
+        self.flags & Self::MAKE_DEFAULT != 0
+    }
+
+    const fn changes_locality(&self) -> bool {
+        self.flags & Self::CHANGE_LOCALITY != 0
+    }
+
+    /// Check if it will be able to create a new default.
```

Comment:
I think it would helpful to expand on this comment a little bit about what "safe" means.

Reply:
I added some more information. Let me know if you think that is enough.

Reply:
That looks good, thanks!